src/redux/news/news.sagas.js

```diff
@@ -82,11 +84,17 @@ function* handleArticleLoad({ payload }) {
     );
     yield put(setArticle(article.data.getNewsById));
     yield put(setLoading(false));
-  } catch (error) {
-    console.log(error);
+  } catch (e) {
+    yield call(handleNewsError, e);
```

Comment:
Why have you created a separate function for error handling if it is used only once?

Reply:
In news.sagas it's used two times - in getAllNews and getNewsById